lms/envs/aws.py

```diff
@@ -896,4 +896,7 @@
 
 ############## Settings for the Enterprise App ######################
 
-ENTERPRISE_ENROLLMENT_API_URL = ENV_TOKENS.get('ENTERPRISE_ENROLLMENT_API_URL', ENTERPRISE_ENROLLMENT_API_URL)
+ENTERPRISE_ENROLLMENT_API_URL = ENV_TOKENS.get(
+    'ENTERPRISE_ENROLLMENT_API_URL',
+    (LMS_ROOT_URL or '') + '/api/enrollment/v1/'
```

Comment:
While that would help readability a bit, I'd rather "leave" it for now to avoid littering settings with variables that aren't used elsewhere.